calcom/cal.com
# Fix - Avatar on New Event Type

## What does this PR do?

When selecting a new event type, if you have an individual plan as well as a team plan. The Team Plan's Avatar shows, but the individual user's avatar does not show. So, this PR fixes this issue. Now if there is no avatar then the default avatars will be shown.

Fixes #3336 

 Loom Video: https://www.loom.com/share/3d21119e4bec498da7135e79c1306c22

## Type of change




## How should this be tested?



- Login with a user who has a team associated with him.
- Make sure that the user doesn't have an avatar and the team has an avatar,
- Navigate to `Event Type` and click on the `New Event Type` dropdown. You will be able to see the avatar of the team and the default avatar for the user.
